docs/developer-docs/latest/getting-started/usage-information.md

```diff
@@ -19,7 +19,7 @@ However, these above actions alone are often insufficient to maintain an overall
 - How long does setting up a project take? If the global install time increases, does it mean that users are encountering issues or the process is simply too complicated?
 - What type of errors our users are facing?
 - What are the most used plugins?
-- Should we focus our efforts to being compatible with Node 12? Maybe our community uses version 12, in greater percentages than the global Node.js community?
+- Should we focus our efforts to being compatible with Node 18? Maybe our community uses version 18, in greater percentages than the global Node.js community?
```

Comment:
We won't touch node 18 until it goes LTS

Reply:
Thank you Derrick. I didn't know how to update this sentence. I assumed it was written before 12 was on LTS. Should I change the text to "version 16"? It felt weird to me because we already support Node 16, right?

Reply:
Yes exactly